content/en/user-guide/user-defined-transaction-management/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "User-Defined Transaction Management"
```

Comment:
It’s better to name it simply `Transaction Management`, as it’s more concise while still clearly conveying the purpose of the page. Please update this in other places as well